fingerprint2.js

```diff
@@ -498,6 +500,56 @@
         done(keys);
       }, 1);
     },
+	audioKey: function(keys, done){
+		if ((!this.options.excludeAudio) && !(("ontouchstart" in window) || (navigator.maxTouchPoints > 0) || (navigator.msMaxTouchPoints > 0))) {
```

Comment:
Sorry forthe late reply,

Checking if it is a mobile/tablet, on my tests the audio fingerprint didn't work well on them.